src/Generator.php

```diff
@@ -1,4 +1,6 @@
 <?php
+declare(strict_types=1);
```

Comment:
This should go below the license header. The file should look like this:

```
<?php
/**
 * <license header goes here>
 */

declare(strict_types=1);

namespace MyNamespace;

// Any use statements would go here

class MyClass
{
}

Reply:
Resolved.